bundles/AdminBundle/Resources/public/js/pimcore/object/classificationstore/keySelectionWindow.js

```diff
@@ -435,7 +435,37 @@ pimcore.object.classificationstore.keySelectionWindow = Class.create({
         this.store = new Ext.data.Store({
             remoteSort: true,
             proxy: proxy,
-            fields: readerFields
+            fields: readerFields,
+            filters: [
+                function(item) {
```

Comment:
can you explain why this is needed and how it works ? don't get the point here especially because it is supposed to return a boolean ?

Reply:
It is used to translate the texts, because I have dropped the changes in the backend.

Filters are used to modify values, you can write a function to process the complete data set or define one filter per data key. This is necessary because the data is passed directly through the API into the GRID table over the proxy.

Well, you know, I missed the return type, and I'll fix that right up.

Reference to the ExtJs documentation:
https://docs.sencha.com/extjs/6.2.0/modern/Ext.data.AbstractStore.html#cfg-filters

Reply:
And this cannot be done in an event handler (e.g. on "load") ?

Reply:
This would also work but I find that filters are better for this.

With "load" I can change the data, but I have to overwrite it in the store with ".bind(this)". Personally, I find this a very bad solution.

If you prefer, I can make that like this way. If it really works in detail. I was just touching on that until I saw the filters.

Reply:
If it doesn't work with event then please add inline documentation explaing why it is done (similar to what you described above). I think otherwise this is not obvious :-)

Reply:
Done, with the load handler 👍